cli/cmd/install.go

```diff
@@ -286,13 +316,35 @@ func render(config installConfig, w io.Writer, options *installOptions) error {
 
 	// Merge templates and inject
 	var buf bytes.Buffer
-	bt := path.Join(renderOpts.ReleaseOptions.Name, baseTemplateName)
-	if _, err := buf.WriteString(renderedTemplates[bt]); err != nil {
+	t := path.Join(renderOpts.ReleaseOptions.Name, nsTemplateName)
```

Comment:
Should this be guarded by `if !config.SingleNamespace` ?  (I'm not really sure what this does or when/why it should be skipped.

Reply:
This is already guarded by the template syntax `{{ if not .Values.SingleNamespace -}}` in `namespace.yaml`. If `SingleNamespace` is not set, then that template will not be rendered.